### ISSUE TITLE
Screen rotation on Win10 tablet

### ISSUE THREAD

**Issue by kenjiuno** [fluentribbon/Fluent.Ribbon#180]
Hi.

I'm trying to use Fluent Ribbon on Windows 10 tablet.

When I rotate the screen, black part appears:
- Vert to horz → black part at right side
- Horz to vert → black part at bottom part

Screenshot:
![change_orientation](https://cloud.githubusercontent.com/assets/5955540/10533052/9e214f1e-73ff-11e5-9bd7-4d21677ec888.jpg)

The visual recovers if I maximize the window again (→restore→maximize).

My xaml:
```xml
<F:RibbonWindow x:Class="W10Fluent.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:F="urn:fluent-ribbon" 
        Title="MainWindow" Height="350" Width="525">
    <Window.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="/Fluent;Component/Themes/Office2013/Generic.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Window.Resources>
    <Grid>
        <Grid.RowDefinitions>
            <RowDefinition Height="auto" />
            <RowDefinition Height="*" />
        </Grid.RowDefinitions>
        <F:Ribbon Grid.Row="0">
            <F:RibbonTabItem Header="Component/Themes/Office2013/Generic.xaml">
                <F:RibbonGroupBox Header="Masters">
                    <F:Button Header="Table xxx" LargeIcon="Resources/Table_32.png" SizeDefinition="Large" />
                    <F:Button Header="Table xxx" LargeIcon="Resources/Table_32.png" SizeDefinition="Large" />
                    <F:Button Header="Table xxx" LargeIcon="Resources/Table_32.png" SizeDefinition="Large" />
                    ...
                </F:RibbonGroupBox>
            </F:RibbonTabItem>
        </F:Ribbon>
    </Grid>
</F:RibbonWindow>
```

Anyone, are there known workarounds?

**Comment by batzen** [comment]
That's not supposed to happen. The WindowChrome class from Microsoft has tons of bugs we have to workaround. I guess this one has to be worked around too (just have a look at the WindowSizing class in this project).
Could you record the window messages sent to the application when you rotate the screen? I got no tablet to do this myself.
To record the window messages you can use Spy++ which should be installed when you install visual studio.

**Comment by kenjiuno** [comment]
Ok, I'll check it in early next week...

**Comment by batzen** [comment]
Maybe this issue iis related to the same issue that occurs when the window is maximised and the taskbar is set to auto hide...

**Comment by kenjiuno** [comment]
I have captured messages using Spy++ 8.00 (option: include same process).

rotating from portrait to landscape:
```
<00001> 00040360 P message:0x0238 [不明] wParam:00000020 lParam:00010099
<00002> 000402E6 S WM_NCHITTEST xPos:640 yPos:400
<00003> 000402E6 R WM_NCHITTEST nHittest:HTCLIENT
<00004> 000402E6 S WM_NCHITTEST xPos:640 yPos:400
<00005> 000402E6 R WM_NCHITTEST nHittest:HTCLIENT
<00006> 000402E6 S WM_SETCURSOR hwnd:000402E6 nHittest:HTCLIENT wMouseMsg:WM_MOUSEMOVE
<00007> 000402E6 R WM_SETCURSOR fHaltProcessing:False
<00008> 000402E6 P WM_MOUSEMOVE fwKeys:0000 xPos:192 yPos:409
<00009> 000402E6 S WM_NCHITTEST xPos:640 yPos:400
<00010> 000402E6 R WM_NCHITTEST nHittest:HTCLIENT
<00011> 000402E6 S WM_NCHITTEST xPos:192 yPos:409
<00012> 000402E6 R WM_NCHITTEST nHittest:HTCLIENT
<00013> 000402E6 S WM_NCHITTEST xPos:640 yPos:400
<00014> 000402E6 R WM_NCHITTEST nHittest:HTCLIENT
<00015> 000402E6 S WM_SETCURSOR hwnd:000402E6 nHittest:HTCLIENT wMouseMsg:WM_MOUSEMOVE
<00016> 000402E6 R WM_SETCURSOR fHaltProcessing:False
<00017> 000402E6 P WM_MOUSEMOVE fwKeys:0000 xPos:640 yPos:400
<00018> 000502D8 P WM_TIMER wTimerID:1 tmprc:00000000
<00019> 00040360 S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00020> 00040360 R WM_DISPLAYCHANGE
<00021> 000203D2 S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00022> 00040362 S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00023> 00040362 R WM_DISPLAYCHANGE
<00024> 000203D2 R WM_DISPLAYCHANGE
<00025> 000400AC S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00026> 000400AC R WM_DISPLAYCHANGE
<00027> 00040364 S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00028> 00040364 R WM_DISPLAYCHANGE
<00029> 000402E6 S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00030> 000402E6 R WM_DISPLAYCHANGE
<00031> 000203D0 S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00032> 000203D0 R WM_DISPLAYCHANGE
<00033> 00060096 S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00034> 00060096 R WM_DISPLAYCHANGE
<00035> 000502CC S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00036> 000502CC R WM_DISPLAYCHANGE
<00037> 000602C6 S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00038> 000602C6 R WM_DISPLAYCHANGE
<00039> 0005035E S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00040> 0005035E R WM_DISPLAYCHANGE
<00041> 000402E0 S WM_DISPLAYCHANGE cBitsPerPixel:32 cxScreen:1280 cyScreen:800
<00042> 000402E0 R WM_DISPLAYCHANGE
<00043> 000502D8 P message:0xC17C [登録された:"DispatcherProcessQueue"] wParam:00000000 lParam:00000000
<00044> 000502D8 P message:0xC17C [登録された:"DispatcherProcessQueue"] wParam:00000000 lParam:00000000
<00045> 000402E6 S WM_GETTEXT cchTextMax:1024 lpszText:0113D40C
<00046> 000402E6 R WM_GETTEXT cchCopied:10 lpszText:0113DD3C ("M")
<00047> 000402E6 S WM_WINDOWPOSCHANGING lpwp:0113EB7C
<00048> 00040360 S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:06CCFBA0
<00049> 00040360 R WM_SETTINGCHANGE
<00050> 00040362 S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:06CCFBA0
<00051> 00040362 R WM_SETTINGCHANGE
<00052> 000402E6 S WM_GETMINMAXINFO lpmmi:0113E5D8
<00053> 000402E6 R WM_GETMINMAXINFO lpmmi:0113E70C
<00054> 000402E6 R WM_WINDOWPOSCHANGING
<00055> 000402E6 S WM_NCCALCSIZE fCalcValidRects:True lpncsp:0113EB1C
<00056> 000402E6 R WM_NCCALCSIZE fuValidRect:WVR_REDRAW | WVR_VALIDRECTS lpncsp:0113EC78
<00057> 00060096 S WM_WINDOWPOSCHANGING lpwp:0113EB7C
<00058> 00060096 S WM_GETMINMAXINFO lpmmi:0113E5D8
<00059> 00060096 R WM_GETMINMAXINFO lpmmi:0113E70C
<00060> 00060096 R WM_WINDOWPOSCHANGING
<00061> 000502CC S WM_WINDOWPOSCHANGING lpwp:0113EB7C
<00062> 000502CC S WM_GETMINMAXINFO lpmmi:0113E7C8
<00063> 000502CC R WM_GETMINMAXINFO lpmmi:0113E8FC
<00064> 000502CC R WM_WINDOWPOSCHANGING
<00065> 000502CC S WM_NCCALCSIZE fCalcValidRects:True lpncsp:0113EB1C
<00066> 000502CC R WM_NCCALCSIZE fuValidRect:0000 lpncsp:0113EC78
<00067> 000402E0 S WM_WINDOWPOSCHANGING lpwp:0113EB7C
<00068> 000402E0 S WM_GETMINMAXINFO lpmmi:0113E5D8
<00069> 000402E0 R WM_GETMINMAXINFO lpmmi:0113E70C
<00070> 000402E0 R WM_WINDOWPOSCHANGING
<00071> 000402E6 S WM_NCPAINT hrgn:00000001
<00072> 000402E6 R WM_NCPAINT
<00073> 000402E6 S WM_ERASEBKGND hdc:15010E06
<00074> 000402E6 R WM_ERASEBKGND fErased:True
<00075> 000402E6 S WM_WINDOWPOSCHANGED lpwp:0113EB7C
<00076> 000402E6 S WM_MOVE xPos:65525 yPos:65525
<00077> 000402E6 R WM_MOVE
<00078> 000402E6 S WM_SIZE fwSizeType:SIZE_MAXIMIZED nWidth:1302 nHeight:762
<00079> 000402E6 S WM_WINDOWPOSCHANGING lpwp:0113E088
<00080> 000402E6 S WM_GETMINMAXINFO lpmmi:0113DAE0
<00081> 000402E6 R WM_GETMINMAXINFO lpmmi:0113DC14
<00082> 000402E6 R WM_WINDOWPOSCHANGING
<00083> 000402E6 S WM_NCCALCSIZE fCalcValidRects:True lpncsp:0113E028
<00084> 000402E6 R WM_NCCALCSIZE fuValidRect:WVR_REDRAW | WVR_VALIDRECTS lpncsp:0113E184
<00085> 000402E6 S WM_NCPAINT hrgn:00000001
<00086> 000402E6 R WM_NCPAINT
<00087> 000402E6 S WM_ERASEBKGND hdc:45010EE3
<00088> 000402E6 R WM_ERASEBKGND fErased:True
<00089> 000402E6 S WM_WINDOWPOSCHANGED lpwp:0113E088
<00090> 000402E6 S WM_MOVE xPos:10 yPos:0
<00091> 000402E6 R WM_MOVE
<00092> 000402E6 S WM_SIZE fwSizeType:SIZE_MAXIMIZED nWidth:1280 nHeight:740
<00093> 000402E6 S WM_WINDOWPOSCHANGING lpwp:0113D598
<00094> 000402E6 S WM_GETMINMAXINFO lpmmi:0113CFF0
<00095> 000402E6 R WM_GETMINMAXINFO lpmmi:0113D124
<00096> 000402E6 R WM_WINDOWPOSCHANGING
<00097> 000402E6 S WM_WINDOWPOSCHANGING lpwp:0113D598
<00098> 000402E6 S WM_GETMINMAXINFO lpmmi:0113CFF0
<00099> 000402E6 R WM_GETMINMAXINFO lpmmi:0113D124
<00100> 000402E6 R WM_WINDOWPOSCHANGING
<00101> 000402E6 S WM_NCPAINT hrgn:00000001
<00102> 000402E6 R WM_NCPAINT
<00103> 000402E6 S WM_ERASEBKGND hdc:28010AF8
<00104> 000402E6 R WM_ERASEBKGND fErased:True
<00105> 000402E6 S WM_WINDOWPOSCHANGED lpwp:0113D598
<00106> 000402E6 S WM_MOVE xPos:0 yPos:0
<00107> 000402E6 R WM_MOVE
<00108> 000402E6 R WM_WINDOWPOSCHANGED
<00109> 00040364 S message:0x0287 [不明] wParam:00000006 lParam:00000000
<00110> 00040364 R message:0x0287 [不明] lResult:00000000
<00111> 000402E6 S WM_STYLECHANGING wStyleType:GWL_EXSTYLE | GWL_STYLE lpss:0113D5AC
<00112> 000402E6 R WM_STYLECHANGING
<00113> 000402E6 S WM_STYLECHANGED wStyleType:GWL_EXSTYLE | GWL_STYLE lpss:0113D5AC
<00114> 000402E6 R WM_STYLECHANGED
<00115> 000402E6 S WM_STYLECHANGING wStyleType:GWL_EXSTYLE | GWL_STYLE lpss:0113D5AC
<00116> 000402E6 R WM_STYLECHANGING
<00117> 000402E6 S WM_STYLECHANGED wStyleType:GWL_EXSTYLE | GWL_STYLE lpss:0113D5AC
<00118> 000402E6 R WM_STYLECHANGED
<00119> 000402E6 S WM_GETTEXT cchTextMax:1024 lpszText:0113C294
<00120> 000402E6 R WM_GETTEXT cchCopied:10 lpszText:0113CBC4 ("M")
<00121> 000402E6 R WM_SIZE
<00122> 000402E6 R WM_WINDOWPOSCHANGED
<00123> 00040364 S message:0x0287 [不明] wParam:00000006 lParam:00000000
<00124> 00040364 R message:0x0287 [不明] lResult:00000000
<00125> 000402E6 S WM_WINDOWPOSCHANGING lpwp:0113E088
<00126> 000402E6 S WM_GETMINMAXINFO lpmmi:0113DAE0
<00127> 000402E6 R WM_GETMINMAXINFO lpmmi:0113DC14
<00128> 000402E6 R WM_WINDOWPOSCHANGING
<00129> 000402E6 S WM_STYLECHANGING wStyleType:GWL_EXSTYLE | GWL_STYLE lpss:0113E09C
<00130> 000402E6 R WM_STYLECHANGING
<00131> 000402E6 S WM_STYLECHANGED wStyleType:GWL_EXSTYLE | GWL_STYLE lpss:0113E09C
<00132> 000402E6 R WM_STYLECHANGED
<00133> 000402E6 S WM_STYLECHANGING wStyleType:GWL_EXSTYLE | GWL_STYLE lpss:0113E09C
<00134> 000402E6 R WM_STYLECHANGING
<00135> 000402E6 S WM_STYLECHANGED wStyleType:GWL_EXSTYLE | GWL_STYLE lpss:0113E09C
<00136> 000402E6 R WM_STYLECHANGED
<00137> 000402E6 S WM_GETTEXT cchTextMax:1024 lpszText:0113CD84
<00138> 000402E6 R WM_GETTEXT cchCopied:10 lpszText:0113D6B4 ("M")
<00139> 000402E6 R WM_SIZE
<00140> 000402E6 R WM_WINDOWPOSCHANGED
<00141> 000502CC S WM_WINDOWPOSCHANGED lpwp:0113EB7C
<00142> 000502CC S WM_SIZE fwSizeType:SIZE_MAXIMIZED nWidth:1280 nHeight:706
<00143> 000502CC R WM_SIZE
<00144> 000502CC R WM_WINDOWPOSCHANGED
<00145> 00040364 S message:0x0287 [不明] wParam:00000006 lParam:00000000
<00146> 00040364 R message:0x0287 [不明] lResult:00000000
<00147> 000203D2 S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00148> 000203D2 S WM_WINDOWPOSCHANGING lpwp:0113E7F0
<00149> 000203D2 R WM_WINDOWPOSCHANGING
<00150> 000203D2 S WM_NCCALCSIZE fCalcValidRects:True lpncsp:0113E790
<00151> 000203D2 R WM_NCCALCSIZE fuValidRect:0000 lpncsp:0113E8EC
<00152> 000203D2 S WM_WINDOWPOSCHANGED lpwp:0113E7F0
<00153> 000203D2 R WM_WINDOWPOSCHANGED
<00154> 000203D2 R WM_SETTINGCHANGE
<00155> 000400AC S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00156> 000400AC R WM_SETTINGCHANGE
<00157> 00040364 S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00158> 00040364 R WM_SETTINGCHANGE
<00159> 000402E6 S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00160> 000402E6 R WM_SETTINGCHANGE
<00161> 000203D0 S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00162> 000203D0 R WM_SETTINGCHANGE
<00163> 00060096 S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00164> 00060096 R WM_SETTINGCHANGE
<00165> 000502CC S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00166> 000502CC S WM_SHOWWINDOW fShow:True fuStatus:0 (ShowWindow が呼び出されました)
<00167> 000502CC R WM_SHOWWINDOW
<00168> 000502CC S WM_WINDOWPOSCHANGING lpwp:0113E704
<00169> 000502CC R WM_WINDOWPOSCHANGING
<00170> 000502CC S WM_WINDOWPOSCHANGING lpwp:0113E704
<00171> 000502CC R WM_WINDOWPOSCHANGING
<00172> 000203D2 S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00173> 000203D2 R WM_ACTIVATEAPP
<00174> 000400AC S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00175> 000400AC R WM_ACTIVATEAPP
<00176> 00040364 S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00177> 00040364 R WM_ACTIVATEAPP
<00178> 000502CC S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00179> 000502CC R WM_ACTIVATEAPP
<00180> 000402E6 S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00181> 000402E6 R WM_ACTIVATEAPP
<00182> 000203D0 S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00183> 000203D0 R WM_ACTIVATEAPP
<00184> 00060096 S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00185> 00060096 R WM_ACTIVATEAPP
<00186> 000602C6 S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00187> 000602C6 R WM_ACTIVATEAPP
<00188> 0005035E S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00189> 0005035E R WM_ACTIVATEAPP
<00190> 000402E0 S WM_ACTIVATEAPP fActive:True dwThreadID:00000000
<00191> 000402E0 R WM_ACTIVATEAPP
<00192> 000502CC S WM_NCACTIVATE fActive:False
<00193> 000502CC R WM_NCACTIVATE fDeactivateOK:True
<00194> 000502CC S WM_ACTIVATE fActive:WA_ACTIVE fMinimized:False hwndPrevious:(null)
<00195> 000502CC R WM_ACTIVATE
<00196> 00040364 S message:0x0287 [不明] wParam:00000017 lParam:000502CC
<00197> 000502CC S WM_IME_SETCONTEXT fSet:1 (LONG)iShow:C000000F
<00198> 00040364 S WM_IME_SETCONTEXT fSet:1 (LONG)iShow:C000000F
<00199> 000400AC S WM_IME_SETCONTEXT fSet:1 (LONG)iShow:C000000F
<00200> 000400AC R WM_IME_SETCONTEXT
<00201> 000502CC S WM_IME_NOTIFY dwCommand:00000002 dwData:00000000
<00202> 00040364 S WM_IME_NOTIFY dwCommand:00000002 dwData:00000000
<00203> 000400AC S WM_IME_NOTIFY dwCommand:00000002 dwData:00000000
<00204> 000400AC R WM_IME_NOTIFY
<00205> 00040364 R WM_IME_NOTIFY
<00206> 000502CC R WM_IME_NOTIFY
<00207> 00040364 R WM_IME_SETCONTEXT
<00208> 000502CC R WM_IME_SETCONTEXT
<00209> 00040364 R message:0x0287 [不明] lResult:00000000
<00210> 000502CC S WM_SETFOCUS hwndLoseFocus:(null)
<00211> 000502CC R WM_SETFOCUS
<00212> 000502CC S WM_NCPAINT hrgn:00000001
<00213> 000502CC R WM_NCPAINT
<00214> 000502CC S WM_ERASEBKGND hdc:45010EE3
<00215> 000502CC R WM_ERASEBKGND fErased:True
<00216> 000502CC S WM_WINDOWPOSCHANGED lpwp:0113E704
<00217> 000502CC R WM_WINDOWPOSCHANGED
<00218> 000502CC S message:0x033F [不明] wParam:00000000 lParam:0113E590
<00219> 000502CC R message:0x033F [不明] lResult:00000001
<00220> 000502CC S WM_SHOWWINDOW fShow:False fuStatus:0 (ShowWindow が呼び出されました)
<00221> 000502CC R WM_SHOWWINDOW
<00222> 000400AC S WM_WINDOWPOSCHANGING lpwp:0113E700
<00223> 000400AC R WM_WINDOWPOSCHANGING
<00224> 00040364 S WM_WINDOWPOSCHANGING lpwp:0113E700
<00225> 00040364 R WM_WINDOWPOSCHANGING
<00226> 000502CC S WM_WINDOWPOSCHANGING lpwp:0113E700
<00227> 000502CC R WM_WINDOWPOSCHANGING
<00228> 000400AC S WM_WINDOWPOSCHANGED lpwp:0113E700
<00229> 000400AC R WM_WINDOWPOSCHANGED
<00230> 000502CC S WM_WINDOWPOSCHANGED lpwp:0113E700
<00231> 000502CC R WM_WINDOWPOSCHANGED
<00232> 000502CC S WM_KILLFOCUS hwndGetFocus:(null)
<00233> 000502CC R WM_KILLFOCUS
<00234> 00040364 S message:0x0287 [不明] wParam:00000018 lParam:000502CC
<00235> 000502CC S WM_IME_SETCONTEXT fSet:0 (LONG)iShow:C000000F
<00236> 00040364 S WM_IME_SETCONTEXT fSet:0 (LONG)iShow:C000000F
<00237> 000400AC S WM_IME_SETCONTEXT fSet:0 (LONG)iShow:C000000F
<00238> 000400AC R WM_IME_SETCONTEXT
<00239> 000502CC S WM_IME_NOTIFY dwCommand:00000001 dwData:00000000
<00240> 00040364 S WM_IME_NOTIFY dwCommand:00000001 dwData:00000000
<00241> 000400AC S WM_IME_NOTIFY dwCommand:00000001 dwData:00000000
<00242> 000400AC R WM_IME_NOTIFY
<00243> 00040364 R WM_IME_NOTIFY
<00244> 000502CC R WM_IME_NOTIFY
<00245> 00040364 R WM_IME_SETCONTEXT
<00246> 000502CC R WM_IME_SETCONTEXT
<00247> 00040364 R message:0x0287 [不明] lResult:00000000
<00248> 000502CC R WM_SETTINGCHANGE
<00249> 000602C6 S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00250> 000602C6 R WM_SETTINGCHANGE
<00251> 0005035E S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00252> 0005035E R WM_SETTINGCHANGE
<00253> 000402E0 S WM_SETTINGCHANGE wFlag:SPI_ICONVERTICALSPACING pszMetrics:0113EB78
<00254> 000402E0 R WM_SETTINGCHANGE
<00255> 000502CC S WM_GETICON fType:True
<00256> 000502CC R WM_GETICON hicon:00000000
<00257> 000602C6 P message:0xC182 [登録された:"MilChannelNotify"] wParam:00000000 lParam:00000000
<00258> 000502CC S WM_GETICON fType:True
<00259> 000502CC R WM_GETICON hicon:00000000
<00260> 000402E6 P message:0xC188 [登録された:"UpdateWindowSettings"] wParam:00000000 lParam:00000000
<00261> 000502CC S WM_GETICON fType:False
<00262> 000502CC R WM_GETICON hicon:00000000
<00263> 000502D8 P message:0xC17C [登録された:"DispatcherProcessQueue"] wParam:00000000 lParam:00000000
<00264> 000402E6 P message:0xC188 [登録された:"UpdateWindowSettings"] wParam:00000000 lParam:00000000
<00265> 000402E6 P message:0xC188 [登録された:"UpdateWindowSettings"] wParam:00000000 lParam:00000000
<00266> 000402E6 P WM_MOUSELEAVE
<00267> 000502CC P message:0xC07F [登録された:"ImmersiveFocusNotification"] wParam:FFFFFFFC lParam:00000000
<00268> 000502CC S WM_GETOBJECT
<00269> 000502CC R WM_GETOBJECT
<00270> 000502CC P message:0xC0E0 [登録された:"TaskbarButtonCreated"] wParam:00000000 lParam:00000000
<00271> 000502D8 P message:0xC17C [登録された:"DispatcherProcessQueue"] wParam:00000000 lParam:00000000
<00272> 000502D8 P message:0xC17C [登録された:"DispatcherProcessQueue"] wParam:00000000 lParam:00000000
<00273> 000502D8 P message:0xC17C [登録された:"DispatcherProcessQueue"] wParam:00000000 lParam:00000000
<00274> 000502D8 P message:0xC17C [登録された:"DispatcherProcessQueue"] wParam:00000000 lParam:00000000
<00275> 000503B0 P message:0x0400 [ユーザー定義:WM_USER+0] wParam:0000BABE lParam:08290980
<00276> 000502D8 P message:0xC17C [登録された:"DispatcherProcessQueue"] wParam:00000000 lParam:00000000
<00277> 000402E6 S WM_GETTEXT cchTextMax:1024 lpszText:0113D58C
<00278> 000402E6 R WM_GETTEXT cchCopied:10 lpszText:0113DEBC ("M")
<00279> 000400AC P WM_TIMER wTimerID:1 tmprc:00000000
```

hwnd `000402E6` is the WPF window.

I'll review the log on my spare time...

**Comment by kenjiuno** [comment]
This can be reproduced on my Windows 8.1 Desktop pc.

Right click at desktop, [Screen resolution], [Orientation], [Landscape]⇔[Portrait]

Don't forget to maximize the WPF window before rotation!

**Comment by batzen** [comment]
I guess all or at least most issues regarding the WindowChrome will be resolved when #185 is done.
Trying your repro on Windows 10 was the first thing i did when you reported the issue, but there i can't reproduce it.

**Comment by kenjiuno** [comment]
Skipping WM_GETMINMAXINFO process in WindowSizing.HwndHook works for me. The black part won't appear.
```
                case Constants.WM_GETMINMAXINFO:
                    //this.FixMinMaxInfo(hWnd, lParam, out handled);
                    break;
```

This issue and above workaround can be reproduced on both FluentTest and my sample on both Win8.1 and Win10.
